docs/core/runtime-config/garbage-collector.md

```diff
@@ -313,18 +313,19 @@ For more information about some of these settings, see the [Middle ground betwee
 
 ### Affinitize ranges
 
-- Specifies the list of processors to use for garbage collector threads.
+- Specifies the list of processors to use for garbage collector threads. This can either 
 - This setting is similar to [System.GC.HeapAffinitizeMask](#affinitize-mask), except it allows you to specify more than 64 processors.
-- For Windows operating systems, prefix the processor number or range with the corresponding [CPU group](/windows/win32/procthread/processor-groups), for example, "0:1-10,0:12,1:50-52,1:70".
+- For Windows operating systems, prefix the processor number or range with the corresponding [CPU group](/windows/win32/procthread/processor-groups), for example, "0:1-10,0:12,1:50-52,1:7". Note that you cannot use this config if you don't actually have more than 1 CPU group. You must use the [Affinitize mask](#affinitize-mask) setting. And the numbers you specify are within that group, which means it cannot be >= 64.
```

Comment:
> Note that you cannot use this config if you don't actually have more than 1 CPU group

Are you sure it doesn't work? I have thought that I have implemented it so that you can pass in all group ids as zeros in such case and it would still work.

Reply:
yes, if there's no CPU groups, `g_nGroups` is 0 so if I specify 0:1-10, it will fail at this point in `GCToOSInterface::ParseGCHeapAffinitizeRangesEntry` -

```cplusplus
    if (group_number >= g_nGroups)
    {
        // Group number out of range
        return false;
    }
```
I also thought this worked but when I tried it didn't...